### PR TITLE
Require update footer

### PR DESCRIPTION
**Pull request by kylezs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,10 +24,11 @@ Please submit any changes as pull requests to the development branch, all change
 ```
 nvm use 11.9.0
 npm install -g quasar-cli
-git clone https://github.com/loki-project/loki-electron-wallet
+git clone https://github.com/loki-project/loki-electron-gui-wallet
 cd loki-electron-wallet
-cp path_to_loki_binaries/lokid bin/
-cp path_to_loki_binaries/loki-wallet-rpc bin/
+// the binaries are what was downloaded in the prequisite step
```
Comment (Mikunj) [inline README.md:29]: `//` comments usually don't work in command line, i'd rather we add an explanation above the code snippet saying `path_to_lokid_binaries` contains what was downloaded in the pre-requisite

File: src-electron/main-process/modules/backend.js [hunk 2]
```diff
@@ -298,13 +301,38 @@ export class Backend {
...
+      const { data } = await axios.get(
+        "https://api.github.com/repos/loki-project/loki-electron-gui-wallet/releases/latest"
+      );
+      // remove the 'v' from front of the version
+      const latestVersion = data.tag_name.substring(1);
+      // can return "major", "minor", "patch"
+      const vSizeDiff = semver.diff(version, latestVersion);
+      const updateAvailable = semver.ltr(version, latestVersion);
+      const majorOrMinor = vSizeDiff === "major" || vSizeDiff == "minor";
+      let updateRequired = false;
```
Comment (Mikunj) [inline src-electron/main-process/modules/backend.js:317]:
```suggestion
      const updateRequired = updateAvailable && majorOrMinor;
```

File: src/store/gateway/state.js [hunk 3]
```diff
@@ -119,5 +119,9 @@ export default {
     bans: [],
     tx_pool_backlog: [],
     service_nodes: []
+  },
+  update_required: {
+    required: false,
+    latest_version: "Unknown"
```
Comment (Mikunj) [inline src/store/gateway/state.js:125]: seems we're not using `latest_version` so we can just make `update_required a boolean value

File: README.md (hunk 1 above)
Comment (Mikunj) [reply README.md:29]: Maybe
- Download latest Lokid binaries and extract to a folder

File: src/store/gateway/state.js (hunk 3 above)
Comment (kylezs) [reply src/store/gateway/state.js:125]: Yeah was more thinking forward on this one, that it could be useful to store.

**Review by Mikunj — APPROVED** [review]

**Pull request merged**